src/T2DMap.cpp

```diff
@@ -1719,9 +1723,9 @@ void T2DMap::paintRoomExits(QPainter& painter, QPen& pen, QList<int>& exitList,
             }
         }
     }
-    QSetIterator<int> itRoom2(pArea->getAreaRooms());
-    while (itRoom2.hasNext()) {
```

Comment:
This iterator was quicker in benchmark testing and Qt's documentation also says that it is slower.

Reply:
Well, which *is* it? :confused:

Reply:
I meant to say that Java-style iterators are slower and STL style proved to be quicker in benchmarking.

Reply:
TBH I thought that was the case, 😀 but your text was unclear. OTOH The STL iterators are more *fragile* - though in this case it shouldn't be an issue for a container of `int`s that are immediately validated.